packages/fuselage/src/components/Button/Button.tsx

```diff
@@ -41,7 +41,7 @@ export const Button = forwardRef(function Button(
     children,
     ...props
   }: ButtonProps,
-  ref: Ref<HTMLElement>
+  ref: Ref<HTMLButtonElement>
```

Comment:
What if `<Box is='a' ref={ref} />`?

Reply:
you're right ✅ 
added ` | HTMLAnchorElement` type to ref and `AllHTMLAttributes<HTMLButtonElement | HTMLAnchorElement>`